### ISSUE TITLE
AWS-Vault clear does not clear `AWS_VAULT` environmental variable

### ISSUE THREAD

**Issue by hunttom** [99designs/aws-vault#905]
- [ x] I am using the latest release of AWS Vault
- [N] I have provided my `.aws/config` (redacted if necessary)
- [N] I have provided the debug output using `aws-vault --debug` (redacted if necessary)

Running `aws-vault clear` does not clear out the `AWS_VAULT` environmental variable.

Steps to reproduce:
1. Run `aws-vault exec <PROFILE> --ecs-server`
2. Run `echo $AWS_VAULT` to see `<PROFILE>`
3. Run `aws-vault clear`
4. Run `echo $AWS_VAULT` (`<PROFILE>` is still set)
5. Validate profile is still set by running `aws-vault exec <PROFILE> --ecs-server`
6. Still get warning: `aws-vault: error: exec: aws-vault sessions should be nested with care, unset AWS_VAULT to force`

Recommend adding to the clear function:
```go
	if os.Getenv("AWS_VAULT") != "" {
		err = os.Unsetenv("AWS_VAULT")
		if err != nil {
			return fmt.Errorf("Unable to unset environmental variable: 'AWS_VAULT'")
		}
	}
```

**Comment by ivankovnatsky** [comment]
I used this piece of nonsense to workaround it: 

```
set | rg AWS_ | rg -v AWS_VAULT_BACKEND | rg -v ZLE | sed 's/=.*//' | while read i; do unset $i; done
```

**Comment by acolominas** [comment]
Same error for me. 
Everytime I need to switch between profiles I need to execute "unset AWS_VAULT" after the commando "aws-vault clear"

**Comment by mtibben** [comment]
When a command isn't specified, aws-vault creates a "subshell" by running `$SHELL`. The latest pre-release makes this clearer. You don't need to clear env vars. Instead, `exit` out of the subshell.

Fixed in #1136
